java/src/org/openqa/selenium/grid/data/DefaultSlotMatcher.java

```diff
@@ -91,6 +92,10 @@ public boolean matches(Capabilities stereotype, Capabilities capabilities) {
     return browserNameMatch && browserVersionMatch && platformNameMatch;
   }
 
+  private Boolean browserVersionMatch(String stereotype, String capabilities) {
```

Comment:
not realy relevant, just as an info: this could be a primitive `boolean`, not an `Boolean`.